java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -181,4 +196,48 @@ public CompletableFuture<String> set(
         String[] arguments = ArrayUtils.addAll(new String[] {key, value}, options.toArgs());
         return commandManager.submitNewCommand(SetString, arguments, this::handleStringOrNullResponse);
     }
+
+    @Override
+    public CompletableFuture<Long> zadd(
+            @NonNull String key,
+            @NonNull Map<String, Double> membersScoresMap,
+            @NonNull ZaddOptions options,
+            boolean changed) {
+        String[] changedArg = changed ? new String[] {"CH"} : new String[] {};
+
+        String[] membersScores =
+                membersScoresMap.entrySet().stream()
+                        .flatMap(e -> Stream.of(e.getValue().toString(), e.getKey()))
+                        .toArray(String[]::new);
```

Comment:
San added something the same - see `convertMapToArgArray`: 
https://github.com/aws/glide-for-redis/pull/955/files#diff-a8496119f8c241080d754c58d737da632800aeb387fc1474754d68d9148a59b5

Reply:
It's similar, but this one swaps the value and key. His implementation also isn't merged yet.